pyproject.toml

```diff
@@ -8,6 +8,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "~3.8"
 fastapi = "0.78.0"
+fastapi-mail = "1.2.2"
```

Comment:
also with `poetry install` we can see similar situation

`Because e-university-api depends on fastapi-mail (1.2.2) which doesn't match any versions, version solving failed.
`

Reply:
fixed